backend/app/cmd/server.go

```diff
@@ -305,6 +305,7 @@ func (s *ServerCommand) newServerApp() (*serverApp, error) {
 		DisqusImporter:    &migrator.Disqus{DataStore: dataService},
 		WordPressImporter: &migrator.WordPress{DataStore: dataService},
 		NativeExporter:    &migrator.Native{DataStore: dataService},
+		NewMapperFn:       migrator.NewUrlMapper,
```

Comment:
Passing NewMapper as a func is a nice idea. The only confusing part (at least for me) is the name. Instead of NewMapperFn I'd suggest something less exotic, like `UrlMapperMaker` wich may communicate the intent better